app/views/providers/proceeding_merits_task/check_who_client_is/show.html.erb

```diff
@@ -7,7 +7,9 @@
     <p class="govuk-body"><%= t(".paragraph") %></p>
   <% end %>
 
-  <%= render("shared/check_answers/relationship_to_child", proceeding: @proceeding, relationship_to_child: @relationship_to_child, read_only: false) %>
+  <%= govuk_summary_list(card: { title: page_title }, actions: true) do |summary_list| %>
+    <%= render("shared/check_answers/relationship_to_child", summary_list:, proceeding: @proceeding, relationship_to_child: @relationship_to_child) %>
```

Comment:
this no longer has the ```, read_only: false``` attribute

Reply:
I think it's because it's not being used in the partial anywhere. And, even though i've changed it, it doesn't look like it was before